lib/interfaces/multer-extended-options.interface.ts

```diff
@@ -13,6 +13,7 @@ export interface MultipleSizeOptions {
 
 export interface MulterExtendedOptions extends Pick<MulterOptions, 'fileFilter' | 'limits'> {
   dynamicPath?: string;
+  dynamicFilename?: boolean;
```

Comment:
How about switching the variable name from `dynamicFilename` to `randomFilename`?
```suggestion
  randomFilename?: boolean;
```